scripts/functions-deploy-tests/tests.ts

```diff
@@ -0,0 +1,328 @@
+import * as path from "node:path";
+import * as fs from "node:fs/promises";
```

Comment:
prefer using `fs-extra` of `fs/promises` (`fs-extra` predated `/promises` and has some nice features - though I guess since this is a standalone test, I could get over either)

Reply:
oh interesting. I would've thought that we would prefer using stdlib.

Switched though I'm not sure exactly what we are getting.